app/models/project.rb

```diff
@@ -1,18 +1,19 @@
 class Project < ActiveRecord::Base
-  
+
   belongs_to :company
   has_many  :quotations
   has_many  :sales_orders
-  has_many  :sales_order_lines, through: :sales_orders, 
-              conditions: "sales_orders.status in ('accepted','invoiced')"  
+  has_many  :sales_order_lines,
+    -> {conditions "sales_orders.status in ('accepted','invoiced')"},
```

Comment:
This line is throwing up an error "undefined method `conditions' for #<ActiveRecord::Relation []> (NoMethodError)". 
Changing it to -> {where "sales... }